shell/command_test.go

```diff
@@ -285,6 +285,23 @@ func TestSelectCustomShell(t *testing.T) {
 	assert.Empty(t, shell)
 }
 
+func TestRunShellWorkingDir(t *testing.T) {
+	buffer := &bytes.Buffer{}
```

Comment:
`buffer := new(strings.Builder)` can be used for `cmd.Stdout` and has `buffer.String()`